docs/ds/wblt.md

```diff
@@ -1,20 +1,20 @@
 author: hsfzLZH1, cesonic
 
-WBLT，全称 Weight Balanced Leafy Tree，一种不常见的平衡树写法，但是具有常数较小，可以当做可并堆使用的优点。
+WBLT, abbreviation for weight balanced leafy tree, is an uncommon way of implementing a balanced tree. However, it has the advantage that its constant is relatively small and can be used as [mergeable heap](https://en.wikipedia.org/wiki/Mergeable_heap).
 
-类似于 WBT（weight-balanced trees），WBLT 体现了 leafy 的性质，即节点多，怎么多呢？
+Similar to WBT (weight-balanced trees), WBLT shows the propertity of leafy, that is, it has many nodes. But how many?
 
-对于 n 个数，不同于 treap 等，WBLT 会建立 2n 个节点，每个节点的权值为其右儿子的权值，且右儿子的权值大于等于左儿子
+For $n$ numbers, unlike [treap](./treap) and others, WBLT will create $2n$ nodes. The weight of each node is the weight of its right child, and weight of the right child is greater than or equal to the left child.
```

Comment:
create 2n nodes with the weight equal to that of its right child, which is greater than or equal to left child?